braid/_braid_status.h

```diff
@@ -103,6 +108,19 @@ _braid_AccessStatusInit(braid_Real          t,                /**< current time
                         braid_AccessStatus  status            /**< structure to initialize */
                         );
 
+/**
+ * Initialize a braid_SyncStatus structure
+ */
+braid_Int
+_braid_SyncStatusInit(braid_Int           iter,             /**< current iteration in XBraid*/
+                      braid_Int           level,            /**< current level in XBraid */
+                      braid_Int           nrefine,          /**< number of refinements done */
+                      braid_Int           gupper,           /**< global size of the fine grid */
+                      braid_Int           done,             /**< boolean describing whether XBraid has finished */
+                      braid_Int           calling_function, /**< from which function are we accessing the vector */
```

Comment:
I think you mean "from which function are we accessing the braid" or "..the app"  @jcc242

Reply:
Fixed in commit b79b82f58b83a235334f913e70f8e415518ab83c